### DISCUSSION TITLE
Auto Pause Issue

### DISCUSSION THREAD

**Discussion by Rom0067** [itzg/docker-minecraft-server#3099] [Q&A]
I recently created a minecraft server and set up autopause.  I thought everything was working as I created this for my kids and dont use it myself.   I was doing some maintenance on my server today and realized minecraft was running and my kids have not been using it.  I crashed the server and attached to the container via compose up.  After 5 mins the server paused however after about 5 more minutes I got this error message: 

[18:51:06] [Server thread/WARN] [minecraft/MinecraftServer]: Can't keep up! Is the server overloaded? Running 284715ms or 5694 ticks behind

I had btop running in a second window and my cpu usage was extremely low and most the time was around 1% on all cores.  After this message popped up the container restarted itself.  This leads me to believe that the container has been running for weeks and never truly pausing.  Its not the end of the world but I would like to be able to free up resources when the server is not an use.  Also my other services are extremely lightweight as to not burn up electricity.  Any help towards a solutions is much appreciated.  Thanks!

**Comment by itzg** [comment]
That warning is normal. When it resumes it sees that a bunch of time has elapsed between ticks. 

> After 5 mins the server paused however after about 5 more minutes I got this error message:

Something might have pinged the port and woke it up.

**Reply by Rom0067** [reply]
Thank you for the reply. I'm not sure what would have pinged the port as only 2 people are using it and neither of them have been playing.  Do you have any suggestions as to how I could find out?

I spent some time monitoring it while I was doing some other stuff and it pauses every 5 minutes for 10 minutes and then starts up.  It does this indefinitely.  I'm not sure what would be pinging it every 10mins but if you have a suggestion as to how to track it down I would greatly appreciate it. Thank you for your time.

**Reply by itzg** [reply]
I suggest using something like https://www.wireshark.org/ to watch for TCP port 25565 traffic.
